src/Codec/Xlsx/Types.hs

```diff
@@ -99,6 +107,11 @@ import Codec.Xlsx.Types.StyleSheet as X
 import Codec.Xlsx.Types.Table as X
 import Codec.Xlsx.Types.Variant as X
 import Codec.Xlsx.Writer.Internal
+#ifdef USE_MICROLENS
+import Lens.Micro
+#else
+import Control.Lens hiding ((.=))
```

Comment:
I'll make these selective, normally I just import everything because I think the convenience of importing everything doesn't way up against the difficulty of an ambiguous function.